Assets/Global/Scripts/Player/States/AttackingState.cs

```diff
@@ -1,11 +1,13 @@
+using System;
+using UnityEngine;
```

Comment:
System and UnityEngine seem to be unused

Reply:
I am merging it now. Salah you can change it in the ticket you are working now right now.
It is indeed nice if we clean things lik this up, but at least this import will not break when compiling a build :)